backend/FactoryOps.Api/Controllers/ItemsController.cs

```diff
@@ -9,31 +10,42 @@ namespace FactoryOps.Api.Controllers;
 public class WorkItemsController(IRepository<Item> itemsRepository) : ControllerBase
 {
 	[HttpGet]
-	public ActionResult<IAsyncEnumerable<Item>> GetWorkItems() => Ok(itemsRepository.GetAll());
-	
+	public ActionResult<IAsyncEnumerable<ItemDto>> GetAll()
+	{
+		return Ok(itemsRepository.GetAll()
+			.Select(item => new ItemDto(
+				item.Id,
+				item.Group,
+				item.Title,
+				item.StartTime.ToString("yyyy-MM-dd:HH:mm:ss"),
```

Comment:
I don't think we need to use seconds when we have elements moved every hour

Reply:
I think we won't use even minutes and hours, but for now, until we receive any feedback, I want to save it.

Reply:
ok